cmd/geth/flags.go

```diff
@@ -152,8 +152,37 @@ var (
 	}
 	LogDirFlag = DirectoryFlag{
 		Name:  "log-dir,logdir",
-		Usage: "Directory in which to write log files.",
-		Value: DirectoryString{filepath.Join(common.DefaultDataDir(), "logs")},
+		Usage: "Directory in which to write log files",
+		Value: DirectoryString{filepath.Join(common.DefaultDataDir(), "<chain>", "logs")},
+	}
+	LogMaxSizeFlag = cli.IntFlag{
+		Name:  "log-max-size,log-maxsize",
+		Usage: "Maximum size of a single log file (in bytes)",
+		Value: 0,
+	}
+	LogMinSizeFlag = cli.IntFlag{
+		Name:  "log-min-size,log-minsize",
+		Usage: "Minimum size of a log file, to be considered for log-rotation (in bytes)",
+		Value: 0,
+	}
+	LogMaxTotalSizeFlag = cli.IntFlag{
+		Name:  "log-max-total-size,log-maxtotalsize",
```

Comment:
I suggest naming this `log-total-max-size,log-totalmaxsize` to stick with the convention established above where `<max|min>-size` is a "unit." What do you think?

Reply:
Fixed.